backend/src/mirrors_qa_backend/entrypoint.py

```diff
@@ -0,0 +1,31 @@
+import argparse
+import logging
+
+from mirrors_qa_backend import Settings, db, logger
+from mirrors_qa_backend.db import mirrors
+from mirrors_qa_backend.extract import get_current_mirrors
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
```

Comment:
Why is this option still present? What purpose does it serve? If it's not passed, the script does nothing, right? So it's useless and should be removed. Either one wants to update mirrors and launches the script or doesn't want to and then does not run the script.
That's why I suggested this script to be _dedicated_. ie have a feature-centric name like `update-mirrors` instead of being “the backend script”